cmd/pctl/main.go

```diff
@@ -84,6 +88,65 @@ func showCmd() *cli.Command {
 	}
 }
 
+func installCmd() *cli.Command {
+	return &cli.Command{
+		Name:      "install",
+		Usage:     "generate configuration objects for later cluster setup",
```

Comment:
this doesn't seem right. Its more like "generate resources for installing a profile" or something along those lines

Reply:
Yeah, I meant something along the way of saying, setup your cluster, like, your kube cluster, set it up with the profile you will be installing. :D It sounds weird, now reading it after the weekend. :D